examples/quickstart_fastai/requirements.txt

```diff
@@ -1,3 +1,2 @@
-fastai~=2.7.12
-flwr~=1.4.0
-torch~=2.0.1
+flwr>=1.0,<2.0
+fastai==2.7.10
```

Comment:
Why not 2.7.12?

Reply:
I'm just imitating the way poetry export handles version with ^; it just fixes the version in requirements.txt with == and omit the ^ entirely. But I'm open to change the version with >= 2.7.10, <3.0.0.

Reply:
@danielnugraha I think what Daniel meant is to use 2.7.12 instead of 2.7.10. So let's update it in `pyproject.toml` and `requirements.txt` to the latest patch version before merging?